From 2d30efbe30ddcdf8ed2a54f9a633e065cb910e2b Mon Sep 17 00:00:00 2001
From: Ke Qinghua <qinghua.ke@freescale.com>
Date: Wed, 19 Mar 2014 12:28:06 +0800
Subject: [PATCH 301/310] ENGR00303453-2:[Android_ MX6DL_ARD]Boot: Android
 cannot boot up from NAND.  100%

Fix SD card auto mount point

Signed-off-by: Ke Qinghua <qinghua.ke@freescale.com>
---
 sabreauto_6q/fstab_nand.freescale | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sabreauto_6q/fstab_nand.freescale b/sabreauto_6q/fstab_nand.freescale
index 385fb17..530d5c7 100644
--- a/sabreauto_6q/fstab_nand.freescale
+++ b/sabreauto_6q/fstab_nand.freescale
@@ -5,7 +5,7 @@
 
 # below is for ubi support, uncomment to support ubifs(NAND)
 
-/devices/platform/sdhci-esdhc-imx.0/mmc_host/mmc1 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
+/devices/platform/sdhci-esdhc-imx.2/mmc_host/mmc0 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
 /devices/platform/fsl-ehci /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
 /dev/ubi0_0     /system		    ubifs 	ro                                      wait
 /dev/ubi0_3     /data           ubifs   nosuid,nodev,nodiratime,noatime         wait
-- 
1.8.0

